pkg/container/container_test.go

```diff
@@ -15,6 +22,83 @@ var _ = t.Describe("Container", func() {
 		}
 		sboxConfig = &pb.PodSandboxConfig{}
 	})
+	t.Describe("AddMountToSpec", func() {
+		It("should add the mount to the spec", func() {
+			sut.AddMountToSpec("test", "test", "test", []string{"test"})
+			Expect(len(sut.GetSpec().Mounts())).To(Equal(7))
+		})
+		It("should add only one copy to the spec", func() {
+			sut.AddMountToSpec("test", "test", "test", []string{"test"})
+			sut.AddMountToSpec("test", "test", "test", []string{"test"})
+			Expect(len(sut.GetSpec().Mounts())).To(Equal(8))
+		})
+	})
+	t.Describe("GetSpec", func() {
+		It("should return the spec", func() {
+			Expect(sut.GetSpec()).ToNot(Equal(nil))
+		})
+	})
+	t.Describe("AddSpecAnnotations", func() {
+		(It("should set the spec annotations", func() {
+
+			// Given
+			sandboxConfig := &pb.PodSandboxConfig{
+				Metadata: &pb.PodSandboxMetadata{Name: "name"},
+			}
+			containerConfig := &pb.ContainerConfig{
+				Metadata: &pb.ContainerMetadata{Name: "name"},
+				Linux: &pb.LinuxContainerConfig{
+					SecurityContext: &pb.LinuxContainerSecurityContext{
+						Privileged: true,
+					},
+				},
+				Image: &pb.ImageSpec{
+					Image: "img",
+				},
+			}
+			err := sut.SetConfig(containerConfig, sandboxConfig)
+			Expect(err).To(BeNil())
+
+			sb, err := sandbox.New("sandboxID", "", "", "", "test",
+				make(map[string]string), make(map[string]string), "", "",
+				&pb.PodSandboxMetadata{}, "", "", false, "", "", "",
+				[]*hostport.PortMapping{}, false)
+			Expect(err).To(BeNil())
+			currentTime := time.Now()
+			volumes := []oci.ContainerVolume{}
+			imageResult := storage.ImageResult{}
+			mountPoint := "test"
+			configStopSignal := "test"
+
+			Expect(currentTime).ToNot(BeNil())
+			Expect(sb).ToNot(BeNil())
+
+			err = sut.AddSpecAnnotations(sb, volumes, mountPoint, configStopSignal, &imageResult, false, false)
+			Expect(err).To(BeNil())
+
+			Expect(sut.GetSpec().Config.Annotations[annotations.Image]).To(Equal("img"))
```

Comment:
for all of these, can we compare against variables instead of hard coding twice

Reply:
fixed!